app/controllers/admin/messages.js

```diff
@@ -0,0 +1,36 @@
+import Controller from '@ember/controller';
+
+export default Controller.extend({
+
+  columns: [
+    {
+      propertyName : 'recipient',
+      title        : 'Recipients'
+    },
+    {
+      propertyName : 'action',
+      title        : 'Trigger'
+    },
+    {
+      propertyName : '',
```

Comment:
@mayank8318 why is the property name blank? 'emailSubject' should be the property name and can directly be used in the hbs file.

Reply:
But can I pass more than one properties?

Reply:
And what you said doesn't work. I will have to do declare something like this ->
```javascript
    {
      emailSubject : this.get('record.emailSubject'),
     emailMessage: this.get('record.emailMessage'),
      title        : 'Email Message',
      template     : 'components/ui-table/cell/admin/messages/cell-email-message'
    }
```

Reply:
Sorry event this doesn't work. Can you give any example where this has been done?

Reply:
No not like this.
```javascript
 {
      propertyName : 'startsAt',
      template     : 'components/ui-table/cell/cell-simple-date',
      dateFormat   : 'MMMM DD, YYYY - HH:mm A',
      title        : 'Date'
    }
```
and then in the template 
```hbs
{{#if (get record column.propertyName)}}
  <span>
    {{moment-format (get record column.propertyName) (if column.dateFormat column.dateFormat 'MMMM DD, YYYY - HH:mm A')}}
  </span>
{{else}}
  <span>
    {{t 'No dates available.'}}
  </span>
{{/if}}
```